builtin/logical/transit/path_restore.go

```diff
@@ -0,0 +1,60 @@
+package transit
+
+import (
+	"encoding/base64"
+
+	"github.com/hashicorp/vault/helper/jsonutil"
+	"github.com/hashicorp/vault/helper/keysutil"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func (b *backend) pathRestore() *framework.Path {
+	return &framework.Path{
+		Pattern: "restore/?$",
```

Comment:
I would make an optional name to allow them to restore to a different name if desired.

Reply:
Fixed.